### PR TITLE
Issue 3: Image Index

### PR DESCRIPTION
**Pull request by chen-daniel**

### CODE REVIEW

**Comment by chen-daniel** [comment]
[Issue 3: Image Index](https://trello.com/c/Fde9hSbA/73-issue-3-image-index)

File: app/views/application/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,13 @@
-<h1>Hello World</h1>
-<a href="<%= new_image_path %>">Upload Image</a>
+<div class="content">
+  <h1>Google-Images-gram</h1>
+  <a href="<%= new_image_path %>">Upload Image</a>
+  <table>
```
Comment (raj-al-ghul) [inline app/views/application/home.html.erb]: ok for now, but table isn't a great way to accomplish this

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,18 @@
  *= require_tree .
  *= require_self
  */
+
```
Comment (raj-al-ghul) [inline app/assets/stylesheets/application.css]: could we separate this into a separate commit?

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
-  def home; end
+  def home
```
Comment (ztmeyer) [inline app/controllers/application_controller.rb]: What we're adding here is really an 'index' for images. Instead of using application home, could we switch to using the existing Image-related scaffolding?

(In this case, we would create a new `index` action in the images controller, create a new index view, add a new route, and add tests.)

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,4 +1,6 @@
...
+    @images = Image.order(id: :desc)
```
Comment (raj-al-ghul) [inline app/controllers/application_controller.rb]: can you think of another attribute that may be more appropriate to order on if we're looking for most recent?

File: test/controllers/application_controller_test.rb [hunk 5]
```diff
@@ -2,8 +2,14 @@
...
   test 'should get home' do
+    Image.create!(url: 'https://www.image.com/image.jpg')
+    Image.create!(url: 'https://www.image.com/image2.jpg')
+    Image.create!(url: 'https://www.image.com/image3.jpg')
     get root_path
     assert_response :ok
-    assert_select 'h1', 'Hello World'
+    assert_select 'img' do |images|
+      assert_equal images.length, Image.count
+      assert_equal images.first.attribute('src').value, 'https://www.image.com/image3.jpg'
```
Comment (raj-al-ghul) [inline test/controllers/application_controller_test.rb]: adding assertions for the other images would make me feel better about the order -- should we include this in the test name / put in separate test? good tests can be great documentation
